docs/quest-definition/example.json

```diff
@@ -319,10 +366,7 @@
           ],
           "quest_answer_dependency": {
             "question_id": 4,
-            "required_value": [
-              "1",
-              "2"
-            ]
+            "required_value": ["1", "2"]
```

Comment:
As you get more into GitHub, here's a friendly PR etiquette tip: I'd avoid mixing cosmetic and content changes in the same PR. The noisy diffs may divert attention from the "meat" of the PR. Good PRs usually have a narrow focus/scope.

In this case, I'm also a little worried that someone may inadvertently reverse the changes with some automated formatting tool&mdash;some editors run these automatically these days.

Reply:
We may want to check in that Prettier config to this repo so tools don't fight with each other over formatting.